### PR TITLE
Total charity distribution

### PR DESCRIPTION
**Pull request by Riglanto**

### CODE REVIEW

File: pages/api/funding/balanceOnDate.ts [hunk 1]
```diff
@@ -1,18 +1,24 @@
...
+} from "@/lib/utils";
 import prisma from "lib/prisma";
 
 export default async function handler(
   req: NextApiRequest,
   res: NextApiResponse
 ) {
   const chains = getChains();
-  const chainsObject: Record<string, any> = chains.reduce(
+  const chainsObject: Record<string, Chain> = chains.reduce(
```
Comment (gglucass) [inline pages/api/funding/balanceOnDate.ts:21]: 🤝

File: pages/api/funding/balanceOnDate.ts [hunk 2]
```diff
@@ -44,10 +50,15 @@ export default async function handler(
     },
   });
 
+  const allocated: { [key: string]: BigNumber } = Object.keys(
```
Comment (gglucass) [inline pages/api/funding/balanceOnDate.ts:53]: should this be a const?

**Review by gglucass — APPROVED** [review]
I <3 it

File: pages/api/funding/balanceOnDate.ts (hunk 2 above)
Comment (Riglanto) [reply pages/api/funding/balanceOnDate.ts:53]: yeah, as we only replace the values under specific keys not the object itself.

**Pull request merged**